charts/opentelemetry-collector/ci/deployment-values.yaml

```diff
@@ -31,3 +31,8 @@ ingress:
         - path: /
           pathType: Prefix
           port: 4318
+
+lifecycleHooks:
+  preStop:
+    sleep:
+      seconds: 10
```

Comment:
This should be an optional feature that users can opt into

Reply:
Makes sense, updated the file.